commercetools-models/src/main/java/io/sphere/sdk/shippingmethods/ShippingMethodDraft.java

```diff
@@ -38,11 +43,21 @@
     @Nullable
     String getPredicate();
 
+    @Deprecated
```

Comment:
Description has been deprecated at the API:

https://docs.commercetools.com/release-notes#releases-2020-03-03-add-localized-shipping-method-description

Reply:
I believe that if the API for the description is not handled anymore  maybe it would be the case to say to our users otherwise they wouldn't notice it

Reply:
But then I would suggest to mark the `description` field as @deprecated too.